tidy3d/components/simulation.py

```diff
@@ -345,6 +352,13 @@ class Simulation(AbstractSimulation):
         All the monitor implementations.
     """
 
+    lumped_elements: Tuple[LumpedElementType, ...] = pydantic.Field(
```

Comment:
so what happens if this is added to the simulation? Does it act as a source? as a medium? I think we need more detailed explanation.